src/main/resources/alma/fix/relatedRessourcesAndLinks.fix

```diff
@@ -152,13 +156,19 @@ do list(path: "830??", "var": "$i")
       copy_field("$i.w", "isPartOf[].$last.hasSuperordinate[].$last.id")
       replace_all("isPartOf[].$last.hasSuperordinate[].$last.id", "^\\(DE-605\\)(.*)$", "http://lobid.org/resources/$1#!")
     end
-    copy_field("$i.a", "isPartOf[].$last.hasSuperordinate[].$last.label")
+    set_array("isPartOf[].$last.hasSuperordinate[].$last.label", " / ")
```

Comment:
Is this intentional, setting the array to `" / "`?

Reply:
Also, are you intentionally not `$append`ing the label here?

Reply:
Thanks, that was an dumb error!